applications/nrf5340_audio/src/bluetooth/bt_management/scanning/Kconfig

```diff
@@ -25,11 +25,16 @@ menu "Connection"
 
 config BLE_ACL_CONN_INTERVAL
 	int "Bluetooth LE ACL Connection Interval (x*1.25ms)"
-	default 32
+	default 64
```

Comment:
Should the default instead be a multiple of both 7.5 ms and 10 ms to be prepared for future use cases?